components/locale_core/src/preferences/extensions/unicode/macros/struct_keyword.rs

```diff
@@ -40,13 +40,29 @@ macro_rules! __struct_keyword {
 
             fn try_from(
                 input: $crate::extensions::unicode::Value,
+            ) -> Result<Self, Self::Error> {
+                Self::try_from(&input)
+            }
+        }
+
+        impl TryFrom<&$crate::extensions::unicode::Value> for $name {
+            type Error = $crate::preferences::extensions::unicode::errors::PreferencesParseError;
+
+            fn try_from(
+                input: &$crate::extensions::unicode::Value,
             ) -> Result<Self, Self::Error> {
                 $try_from(input)
             }
         }
 
         impl From<$name> for $crate::extensions::unicode::Value {
             fn from(input: $name) -> $crate::extensions::unicode::Value {
+                (&input).into()
+            }
+        }
+
+        impl From<&$name> for $crate::extensions::unicode::Value {
```

Comment:
Not sure we need the other direction because these are always `Copy` right? Or not?

Reply:
https://docs.rs/icu/latest/icu/locale/preferences/extensions/unicode/keywords/struct.DictionaryBreakScriptExclusions.html